reviewrot-cronjob-template.yaml

```diff
@@ -0,0 +1,102 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: code-review-reminder-cronjob-template
+  annotations:
+    openshift.io/display-name: "Code review reminder cronjob"
+    description: "Description"
+    tags: "reviewrot code-review-reminder cronjob"
+message: |
+  Cronjob for Code review reminder will run by a schedule ${SCHEDULE}
+  with secrets ${SECRET_CONFIG}/${CONFIG} and
+  ${SECRET_CERTIFICATE}/${CERTIFICATE} on the image
+  quay.io/pbortlov/reviewrot:latest.
+parameters:
+- description: Owner login
+  name: OWNER
+  required: true
+- description: CPU limit
+  name: CPU_LIMIT
+  required: true
+  value: 200m
+- description: Memory limit
+  name: MEMORY_LIMIT
+  required: true
+  value: 256Mi
+- description: Name of configuration's secret
+  name: SECRET_CONFIG
+  required: true
+- description: Name of reviewrot config file
+  name: CONFIG
+  required: true
+- description: Name of certificate's secret
+  name: SECRET_CERTIFICATE
+  required: true
+- description: Name of certificate file
+  name: CERTIFICATE
```

Comment:
Which certificate? CA? I'm thinking that since the CA cert is always the same, could it be part of the template (I mean the entire secret resource object) instead of requiring everyone to upload the same file over again?

Reply:
Certificates are removed

Reply:
So where does it get them now? Are they part of the image?